src/externals/crypto/crypto.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { CryptoService } from './crypto';
+
+describe('Crypto', () => {
+  describe('check crypto as singleton', () => {
+    it('create 2 instances and are the same', () => {
+      const firstInstance = CryptoService.getInstance();
+      const secondInstance = CryptoService.getInstance();
+      expect(firstInstance === secondInstance).toBe(true);
```

Comment:
Is `toBe()` enough to deeply check object equality?

Reply:
sure, you can use toBe to validate value directly:
https://jestjs.io/es-ES/docs/expect#tobevalue